manifests/giantswarm/additional-testing-resources/resume-collection-kustomization.yaml

```
@@ -11,12 +11,13 @@ spec:
       template:
         spec:
           containers:
-            - args:
-                - resume
-                - kustomization
-                - collection
-                - --namespace
-                - flux-giantswarm
+            - command:
+                - sh
+                - -c
+                - |
+                  FLAGS="--namespace flux-giantswarm --verbose"
+                  flux resume kustomization collection $FLAGS
+                  flux resume kustomization collection $FLAGS
```

```suggestion
                  flux resume kustomization flux $FLAGS
```

@ljakimczuk Actually shouldn't this be the flux kustomization?

---

ah, of course it should, stupid mistake, thanks @rossf7 😄